Library/Formula/clisp.rb

```diff
@@ -29,7 +39,7 @@ class Clisp < Formula
 
   def install
     ENV.deparallelize # This build isn't parallel safe.
-    ENV.O0 # Any optimization breaks the build
+    ENV.no_optimization # Build itself sets optimisation flags.
```

Comment:
If that's true, is it safe for us to just remove this entirely? Otherwise we might override it. `no_optimization` is an alias for `O0`.

Reply:
No, because if you don't state anything, -mcpu & -Os will be specified, right?

Reply:
`no_optimization` doesn't strip all optimization flags from the CFLAGS; it sets `O0` explicitly.

Reply:
No, actually, I'm totally wrong. For some reason I remembered otherwise!